### PR TITLE
created table-data

### PR DESCRIPTION
**Pull request by pichar376**

### CODE REVIEW

File: src/buttonDelivery.styles.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const buttonDetailStyle = {
+  color: "#000",
+  textTransform: "none",
+  boxShadow: `0px 1px 4px  rgba(0, 0, 0, 0.15)`,
+  minWidth: 122,
+  minHeight: 40,
+
```
Comment (r3ivaj) [inline src/buttonDelivery.styles.js:7]: Usually you won't need to define the width and the height for a button, you should only care about the padding and the font size and the width and the height will be calculated automatically

File: src/components/ButtonDetails/ButtonDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          fontWeight: "500",
+          fontFamily: "'Inter', sans-serif",
+          fontSize: 16,
+        }}
+      >
+        Details
+      </Typography>
+    </Button>
+  );
+};
```
Comment (r3ivaj) [inline src/components/ButtonDetails/ButtonDetails.jsx:21]: Please remember that you don't need to put a <Typography /> component inside a button, that's an antipattern. You should only have a string as children

File: src/buttonDelivery.styles.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const buttonDetailStyle = {
+  color: "#000",
+  textTransform: "none",
+  boxShadow: `0px 1px 4px  rgba(0, 0, 0, 0.15)`,
+  minWidth: 122,
+  minHeight: 40,
```
Comment (r3ivaj) [inline src/buttonDelivery.styles.js:6]: Usually you won't need to define the height for a button, you should only care about the padding and the font size. The height will be calculated automatically

File: src/components/Menu/Menu.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            fontWeight: "500",
+            fontFamily: "'Inter', sans-serif",
+            fontSize: 16,
+          }}
+        >
+          Actions
+        </Typography>
+      </Button>
+      <StyledMenu
+        id="demo-customized-menu"
```
Comment (r3ivaj) [inline src/components/Menu/Menu.jsx]: this id is not needed
```suggestion
```

File: src/components/Menu/Menu.jsx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        endIcon={<KeyboardArrowDownIcon style={{ color: "#00000075" }} />}
+      >
+        <Typography
+          sx={{
+            color: "#000",
+            fontWeight: "500",
+            fontFamily: "'Inter', sans-serif",
+            fontSize: 16,
+          }}
+        >
```
Comment (r3ivaj) [inline src/components/Menu/Menu.jsx]: Same as above, don't put the typography component inside the button. You should rather style the button component. What if the we want to change the font size for the buttons, will you change the font on every single place?

File: src/components/TableData/Table-data.jsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                </Box>
+              </TableCell>
+              <TableCell>
+                <Box sx={{ display: "flex", flexFlow: "column" }}>
+                  <Typography sx={labelStyle}>Drone</Typography>
+                  <Typography sx={dataStyle}>{row.drone}</Typography>
+                </Box>
+              </TableCell>
+              <TableCell>
+                <Box sx={{ display: "flex", flexFlow: "column" }}>
```
Comment (r3ivaj) [inline src/components/TableData/Table-data.jsx]: If you need to repeat this style probably you can store it in a constant and reuse it

`const nameThisConstant = { display: "flex", flexFlow: "column" }`

File: src/components/TableData/labelStyle.style.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
+  fontSize: 14,
+}
+export const dataStyle = {
+  color: "#000",
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
+  fontSize: 16,
+}
```
Comment (r3ivaj) [inline src/components/TableData/labelStyle.style.js]: please add the new line

File: src/components/TableData/labelStyle.style.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+export const labelStyle = {
+  color: "#00000050",
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
+  fontSize: 14,
+}
+export const dataStyle = {
+  color: "#000",
+  fontWeight: "400",
+  fontFamily: "'Inter', sans-serif",
```
Comment (r3ivaj) [inline src/components/TableData/labelStyle.style.js]: The font family should be global, so that you can avoid having to write it every time you need it

**Comment by r3ivaj** [comment]
Screenshot 2022-11-24 at 9 29 39 p m
Please review the design in figma, your design doesn't match perfectly with the design. 

Be more pixel perfect

File: src/components/ButtonDetails/ButtonDetails.jsx (hunk 2 above)
Comment (pichar376) [reply src/components/ButtonDetails/ButtonDetails.jsx:21]: ok well.

File: src/buttonDelivery.styles.js (hunk 3 above)
Comment (pichar376) [reply src/buttonDelivery.styles.js:6]: ok good to know.

File: src/components/Menu/Menu.jsx (hunk 5 above)
Comment (pichar376) [reply src/components/Menu/Menu.jsx]: Good.

File: src/components/Menu/Menu.jsx (hunk 4 above)
Comment (pichar376) [reply src/components/Menu/Menu.jsx]: Ok

File: src/components/TableData/Table-data.jsx (hunk 6 above)
Comment (pichar376) [reply src/components/TableData/Table-data.jsx]: ok I'll do it.

File: src/components/TableData/labelStyle.style.js (hunk 8 above)
Comment (pichar376) [reply src/components/TableData/labelStyle.style.js]: Ok well.

File: src/components/TableData/labelStyle.style.js (hunk 7 above)
Comment (pichar376) [reply src/components/TableData/labelStyle.style.js]: Ohh yes.

File: src/buttonDelivery.styles.js (hunk 3 above)
Comment (r3ivaj) [inline src/buttonDelivery.styles.js:6]: this hasn't been resolved, the button doesn't need a min-height nor a height, the height is calculated automatically with the padding and the font
```suggestion

```

File: src/components/Button/Button.jsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+import MuiButton from "@mui/material/Button";
+
+const Button = styled(MuiButton)(() => ({
+  textTransform: "none",
+  fontSize: "1rem",
+  boxShadow: "0px 1px 4px rgba(0, 0, 0, 0.1)",
+  border: "1px solid rgba(0, 0, 0, 0.15);",
+  whiteSpace: "nowrap",
+  "&.MuiButton-sizeMedium": {
+    height: "2.5rem",
```
Comment (r3ivaj) [inline src/components/Button/Button.jsx:11]: same as above, the buttons shouldn't have a height as it should be calculated automatically

File: src/globalThem.js [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+      'Arial',
+      'sans-serif'
+    ].join(','),
+  },
+  overries: {
+
+  }
+});
+
+export default theme;
```
Comment (r3ivaj) [inline src/globalThem.js:22]: new line is needed here

File: src/components/ButtonDetails/ButtonDetails.jsx [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+        fontWeight: "500",
+        fontSize: 16,
+      }}
+    >
+      Details
+    </Button>
+  );
+};
+
+export default ButtonDetails;
```
Comment (r3ivaj) [inline src/components/ButtonDetails/ButtonDetails.jsx:23]: Wouldn't it be better to make this a generic/shared button and reuse it across the application?
This is the same as de Actions button. The only thing that changes is the text.

File: src/components/MenuDelivery/MenuDelivery.jsx [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+        onClick={handleClick}
+        endIcon={<KeyboardArrowDownIcon style={{ color: "#00000075" }} />}
+      >
+        Actions
+      </Button>
+      <StyledMenu
+        id="demo-customized-menu"
+        MenuListProps={{
+          "aria-labelledby": "demo-customized-button",
+        }}
```
Comment (r3ivaj) [inline src/components/MenuDelivery/MenuDelivery.jsx:87]: these first two props are not needed for now, maybe you forgot to delete them

File: src/components/Search/Search.jsx [hunk 13]
```diff
@@ -1,19 +1,24 @@
...
           Delivery
         </Typography>
-        <Typography sx={{ color: "#00000060", fontSize: "30px" }}>
+        <Typography
+          sx={{
+            color: "#00000060",
+            fontSize: "30px",
+            fontWeight: 300,
+          }}
+        >
```
Comment (r3ivaj) [inline src/components/Search/Search.jsx:21]: I don't think this component should be called "Search" as this also contains a title. 
It's fine for now if you don't want to change it

File: src/components/Search/components/DeliveryButton/DeliveryButton.jsx [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+        ml: "1rem",
+        textTransform: "none",
+        width: 132,
+        backgroundColor: "#307460",
+        padding: "8px 16px",
+      }}
+    >
+      New Delivery
+    </Button>
+  );
```
Comment (r3ivaj) [inline src/components/Search/components/DeliveryButton/DeliveryButton.jsx:18]: For some reason this button is blue on hover. Maybe it should be the same green color but darkened

**Review by r3ivaj — APPROVED** [review]

File: src/buttonDelivery.styles.js (hunk 3 above)
Comment (pichar376) [reply src/buttonDelivery.styles.js:6]: ok

File: src/components/Button/Button.jsx (hunk 9 above)
Comment (pichar376) [reply src/components/Button/Button.jsx:11]: yes.

File: src/globalThem.js (hunk 10 above)
Comment (pichar376) [reply src/globalThem.js:22]: good

File: src/components/Search/Search.jsx (hunk 13 above)
Comment (pichar376) [reply src/components/Search/Search.jsx:21]: ok good, I will consider it for the future

File: src/components/ButtonDetails/ButtonDetails.jsx (hunk 11 above)
Comment (pichar376) [reply src/components/ButtonDetails/ButtonDetails.jsx:23]: ok.

**Comment by pichar376** [comment]
thanks

File: src/components/Search/components/DeliveryButton/DeliveryButton.jsx (hunk 14 above)
Comment (pichar376) [reply src/components/Search/components/DeliveryButton/DeliveryButton.jsx:18]: ok well.

**Pull request merged**